src/tunacode/ui/commands/clear.py

```
@@ -0,0 +1,51 @@
+
+if TYPE_CHECKING:
+    from tunacode.ui.app import TextualReplApp
+
+
+class ClearCommand(Command):
+    """Clear transient agent state used during command execution."""
+
+    name = "clear"
+    description = "Clear agent working state (UI, thoughts)"
+
+    async def execute(self, app: TextualReplApp, args: str) -> None:
```

Ruff (0.15.0)

[warning] Unused method argument: `args`

(ARG002)